editions/tw5.com/tiddlers/concepts/DefaultTiddlers.tid

```diff
@@ -1,10 +1,15 @@
 created: 20180306113011255
-modified: 20180306161225140
+modified: 20241104150049419
 tags: Concepts
 title: DefaultTiddlers
 type: text/vnd.tiddlywiki
 
-"Default tiddlers" refers to the list of tiddlers that is opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]]).
+"Default tiddlers" refers to the list of tiddlers that is opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]].
+
+There are 2 ways default tiddlers can be defined:
+
+* A [[title-list|Title List]] eg: `TiddlerTitles` and `[[Titles with spaces]]`
+* Filter expressions, using [[filter operators|Filter Operators]] eg: `[tag[HelloThere]]`
```

Comment:
The trouble with this framing is that it misses that title lists are a subset of filter expressions.

Reply:
That's why I did link to the "Title List" tiddler, which explains that fact. For the information here it is "a partial truth" that should be enough to understand that there are more possibilities.